src/lab10.py

```diff
@@ -11,7 +11,6 @@
 import urllib.parse
 import dukpy
 from lab2 import WIDTH, HEIGHT, HSTEP, VSTEP, SCROLL_STEP
-from lab3 import FONTS, get_font
```

Comment:
I think we should keep these imports around—one day I'll fix the inlining system so we don't need them but for now please keep them.

Reply:
Ok, but can you tell me why we need them for the inliner? I didn't know about this (or forgot).

Reply:
I left FONTS but not get_font, which I don't think is needed?

Reply:
I am guessing that `BlockLayout` still calls its `font` method which in turn still calls `get_font`, which we still need.